### PR TITLE
git-fork | update to match REST-API auth standards as of 2021-06

### PR DESCRIPTION
**Pull request by duckunix**
- Removed prompt for OATH Token
- Change prompt from 'password' to 'GitHub Personal Access Token'

### CODE REVIEW

File: bin/git-fork [hunk 1]
```diff
@@ -14,10 +14,8 @@ test -z "$url" && abort "github repo needs to be specified as an argument"
 echo "Enter your github username"
 read user
 [ -n "$user" ] || abort "git username required"
-echo "Enter github password for user \"$user\""
+echo "Enter github Personal Access Token for user \"$user\""
```
[the remark this answers is not in the document]
Comment (bl-ue) [reply bin/git-fork:17]: Oh, interesting 🤔

[the remark this answers is not in the document]
Comment (duckunix) [reply bin/git-fork]: Okay, I have updated the code based on @spacewander 's feedback.  Thanks!

File: bin/git-fork [hunk 2]
```diff
@@ -14,10 +14,12 @@ test -z "$url" && abort "github repo needs to be specified as an argument"
...
 read user
 [ -n "$user" ] || abort "git username required"
-echo "Enter github password for user \"$user\""
-read -s passwd
-echo "Enter github two-factor authentication code (leave blank if not set up)"
-read MFA_CODE
+# personal access token
+# config name is github-personal-access-token '_' is not allowed in git config
+
+github_personal_access_token=$(git config git-extras.github-personal-access-token)
```
Comment (spacewander) [inline bin/git-fork:20]: Please update the doc like this:
https://github.com/tj/git-extras/blob/master/man/git-pull-request.md#description

Comment (duckunix) [reply bin/git-fork:20]: Okay...updated.

**Comment by duckunix** [comment]
This address #927

File: man/git-fork.md [hunk 3]
```diff
@@ -21,6 +21,16 @@ git-fork(1) -- Fork a repo on github
 
   Remotes will use ssh if you have it configured with GitHub, if not, https will be used.
 
+  Create pull request for a project on GitHub via command line.
```
Comment (spacewander) [inline man/git-fork.md:24]: Should update the "pull request" to "fork"

**Review by spacewander — CHANGES_REQUESTED** [review]

File: man/git-fork.md (hunk 3 above)
Comment (duckunix) [reply man/git-fork.md]: Ack...good catch.  Fixed.

File: man/git-fork.md [hunk 4]
```diff
@@ -21,9 +21,9 @@ git-fork(1) -- Fork a repo on github
 
   Remotes will use ssh if you have it configured with GitHub, if not, https will be used.
 
-  Create pull request for a project on GitHub via command line.
+  Create a fork of a project on GitHub via command line.
   
-  A personal access token is required for making the API call to open the pull request(s) in GitHub. [API Documentation here](https://docs.github.com/en/rest/reference/pulls#create-a-pull-request)
+  A personal access token is required for making the API call to create a fork in GitHub. [API Documentation here](https://docs.github.com/en/rest/reference/pulls#create-a-pull-request)
```
Comment (spacewander) [inline man/git-fork.md:26]: The link should be updated.

**Review by bl-ue — APPROVED** [review]

File: man/git-fork.md (hunk 4 above)
Comment (duckunix) [reply man/git-fork.md]: Sorry, missed that.  Thanks for keeping a good eye on the PRs.  Fixed this via 30383f9

**Review by spacewander — APPROVED** [review]

**Pull request merged**